pytest_selenium/pytest_selenium.py

```diff
@@ -116,14 +116,13 @@ def driver_kwargs(request, capabilities, chrome_options, driver_args,
         capabilities=capabilities,
         chrome_options=chrome_options,
         driver_args=driver_args,
-        driver_log=driver_log,
         driver_path=driver_path,
         firefox_options=firefox_options,
         firefox_profile=firefox_profile,
         host=pytestconfig.getoption('host'),
         port=pytestconfig.getoption('port'),
+        service_log_path=driver_log,
```

Comment:
The idea here is that `driver_log` gets used within our `driver_kwargs` methods to update `service_log_path`, which should default to `None`.